src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        this.color = color;
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, color: " + getColor() + ", area: "
```

Comment:
Avoid using direct calls to 'System.out.println' in the 'draw' method. It's better to return a String representation of the object and then print it outside of this class. This approach separates the object's representation from the output method and makes the code more versatile.